omnibus/package-scripts/updater-deb/postinst

```diff
@@ -136,6 +136,7 @@ fi
 chmod 750 ${UPDATER_HELPER}
 setcap cap_setuid+ep ${UPDATER_HELPER}
 
+$INSTALL_DIR/bin/updater/updater bootstrap -P datadog-updater
 $INSTALL_DIR/bin/updater/updater bootstrap -P datadog-agent
```

Comment:
Shouldn't the `datadog-agent` bootstrap be removed? (or is it temporary for now and it will be removed later?)